engine/rotationmanager/lookforwork.go

```diff
@@ -0,0 +1,63 @@
+package rotationmanager
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/riverqueue/river"
+	"github.com/target/goalert/gadb"
+)
+
+type LookForWorkArgs struct{}
+
+func (LookForWorkArgs) Kind() string { return "rotation-manager-lfw" }
+
+// lookForWork will schedule jobs for rotations in the entity_updates table.
+func (db *DB) lookForWork(ctx context.Context, j *river.Job[LookForWorkArgs]) error {
+	var hadWork bool
+	err := db.lock.WithTxShared(ctx, func(ctx context.Context, tx *sql.Tx) error {
+		g := gadb.New(tx)
+
+		rotations, err := g.RotMgrFindWork(ctx)
+		if errors.Is(err, sql.ErrNoRows) {
+			// done, no more work
```

Comment:
```suggestion
			// job's done
```

![image](https://github.com/user-attachments/assets/2e74e50c-4592-4de3-991c-423fe9e69caf)